src/main/java/com/iota/iri/network/TransactionRequesterWorker.java

```diff
@@ -1,32 +0,0 @@
-package com.iota.iri.network;
-
-/**
- * <p>
- * Creates a background worker that tries to work through the request queue by sending random tips along the requested
- * transactions.
- * </p>
- * <p>
- * This massively increases the sync speed of new nodes that would otherwise be limited to requesting in the same rate
- * as new transactions are received.
- * </p>
- */
-public interface TransactionRequesterWorker {
-    
-    /**
-     * Works through the request queue by sending a request alongside a random tip to each of our neighbors.
-     * 
-     * @return <code>true</code> when we have send the request to our neighbors, otherwise <code>false</code>
-     */
-    boolean processRequestQueue();
-
-    /**
-     * Starts the background worker that automatically calls {@link #processRequestQueue()} periodically to process the
-     * requests in the queue.
-     */
-    void start();
-
-    /**
-     * Stops the background worker that automatically works through the request queue.
-     */
-    void shutdown();
-}
```

Comment:
Yay!!
Code is gone :-)

Reply:
![](https://media.giphy.com/media/5xaOcLwEvFOizxHVyVy/giphy.gif)